_inc/client/plans/single-product-backup.jsx

```diff
@@ -158,6 +159,23 @@ class SingleProductBackupBody extends React.Component {
 		this.props.setSelectedBackupType( event.target.value );
 	};
 
+	handleUpgradeButtonClick = selectedBackupType => () => {
+		analytics.tracks.recordJetpackClick( {
+			target: `upgrade-${ selectedBackupType }`,
+			type: 'upgrade',
+			product: selectedBackupType,
```

Comment:
The product here just says `real-time` which is not exactly what we want. I was expecting to see 
`jetpack-backup-realtime` here instead. It helps when the product slug is consistant with other place.

Reply:
We don't have these here.... It will make the component very ugly.

Reply:
Happy do it if you see the value though 👍

Reply:
But if you ask me, whether it’s `real-time` or the actual product slug, it’s all the same. The monthly/yearly views aren’t properly supported here yet, so there’s not a big difference in the current state of the components.